### ISSUE TITLE
CGIAR elevation data not downloaded correctly

### ISSUE THREAD

**Issue by reezer** [graphhopper/graphhopper#2642]
When running the default GraphHopper 5.3 jar using the example config, only changing the elevation provider to srtm and using the web interface the query the elevation graph will be displayed correctly. When using the `cgiar` or `multi` setting no elevation data will be returned.

I downloaded the regular [graphhopper-web-5.3.jar](https://github.com/graphhopper/graphhopper/releases/download/5.3/graphhopper-web-5.3.jar) and used liechtenstein.osm.pbf as a small example. Only the elevation provider was changed (`graph.elevation.provider: cgiar`) compared to the regular config-example.yml.

```
java -Ddw.graphhopper.datareader.file=liechtenstein-latest.osm.pbf -jar graphhopper-web-5.3.jar server config-example.yml
```

This was tested on both Linux and FreeBSD systems. The behavior is identical. The empty /tmp/cgiar will be created, but nothing more. 

OpenJDK was used:
```
Picked up _JAVA_OPTIONS: -Dawt.useSystemAAFontSettings=gasp
openjdk version "18.0.2" 2022-07-19
OpenJDK Runtime Environment (build 18.0.2+9)
OpenJDK 64-Bit Server VM (build 18.0.2+9, mixed mode
```

[gh-test.log](https://github.com/graphhopper/graphhopper/files/9252929/gh-test.log)

**Comment by easbar** [comment]
Same here. I also tried with a few other (small) European extracts: Malta, Georgia, Andorra, but in the maps UI all elevations are zero. What did work though is putting the cgiar file into the cgiar folder manually and then running the import again. In your config file set `graph.elevation.cache_dir: cgiar` and put these two files into the cgiar folder:

https://srtm.csi.cgiar.org/wp-content/uploads/files/srtm_5x5/TIFF/srtm_38_03.zip
https://srtm.csi.cgiar.org/wp-content/uploads/files/srtm_5x5/TIFF/srtm_39_03.zip

You probably only need one of the two for Liechtenstein, but I wasn't sure which one it is. You can find these files here: https://srtm.csi.cgiar.org/srtmdata/

But I'm not sure why it does not work without this workaround...

**Comment by easbar** [comment]
The problem also becomes visible by running the main method in CGIARProvider.java. All the elevations that are printed there are zero.

**Comment by easbar** [comment]
Apparently the download fails with this error:

```
PKIX path validation failed: java.security.cert.CertPathValidatorException: validity check failed
```

during the SSL handshake and we simply return 0 ('sealevel') without even printing a warning if the download fails :roll_eyes: 

The reason seems to be that cgiar's SSL certificate has expired, I also get a browser warning when I try to open their website: https://srtm.csi.cgiar.org/srtmdata/

**Comment by reezer** [comment]
I wrote their site contact an email about this. Would be nice though if GH didn't silently fail in such situations.

**Comment by reezer** [comment]
Update. It was fixed, but maybe this ticket should still stay open for logging or any kind of indication that elevation won't work, should something like this happen again?

**Comment by easbar** [comment]
Nice, thanks! Yes I also think there should be an error or at least a warning. The tricky thing might be that sometimes the elevation is 'missing' only for a few tiles (e.g. where there is only an ocean).

@karussell @boldtrn maybe?

**Comment by boldtrn** [comment]
Yes I agree, we should probably at least log an error if there is an error other than a 404 (which would be for missing elevation data).
